src/pages/components/LetterForm.tsx

```diff
@@ -1,5 +1,5 @@
 import * as Dialog from "@radix-ui/react-dialog";
-import { SubmitLetterMetadata } from "../App";
+import { SubmitLetterMetadata } from "../Home";
 import { Letter } from "./Letter";
 
 export function LetterFormButton() {
```

Comment:
Can't seem to comment on the empty file but `src/pages/components/Profile.tsx` would render a `components/Profile` page I think? Could probably keep it under `src/components/Profile`

Reply:
ohhh true, i should probably move all these then huh